opentech/apply/funds/blocks.py

```diff
@@ -39,6 +39,7 @@ class RichTextFieldBlock(TextFieldBlock):
     widget = TinyMCE(mce_attrs={
         'elementpath': False,
         'branding': False,
+        'toolbar1': 'undo redo | styleselect | bold italic | bullist numlist | link'
```

Comment:
the style select still includes all the possible options.
May be worth using https://www.tinymce.com/docs/configure/content-formatting/#style_formats

Reply:
Great link, thanks. Couldn't find that before.